index.html

```diff
@@ -79,28 +79,26 @@ <h1>Accessibility Map for COVID-19 assessment centres</h1>
 
             <!-- A11y filter panel -->
             <div class="fl-mapviz-filter-panel fl-mapviz-hidden-on-mobile">
-                <div class="fl-mapviz-expand-collapse-button-wrapper">
-                    <button type="button" class="fl-mapviz-expand-collapse-button" aria-expanded="true" aria-label="accessibility filter">
-                        <svg class="fl-mapviz-collapse-button">
-                            <use xlink:href="#ArrowDown"></use>
-                        </svg>
-                        <svg class="fl-mapviz-expand-button">
-                            <use xlink:href="#ArrowUp"></use>
-                        </svg>
-                    </button>
-                </div>
+                <button type="button" class="fl-mapviz-expand-collapse-button" aria-expanded="true" aria-label="accessibility filter">
+                    <svg class="fl-mapviz-collapse-button">
```

Comment:
Sorry I didn't notice this the last time around, but there is a graphic in the grey bar expand/collapse button that results in a "missing label" error.  As elsewhere, it should also have `role="presentation"`.

It's possible there are other graphics that are presentation and should be checked.